### PR TITLE
MS-917 fix: update answers on confirm page

### PR DESCRIPTION
**Pull request by jjrobothamho**
Format session values for display on confirm page

Update transform-data.js
so PDF handles formatted data

### CODE REVIEW

File: apps/nrm/behaviours/format-answers.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  if (req.sessionModel.get('pv-under-age-at-time-of-exploitation')) {
+    req.sessionModel.set(
+      'pv-under-age-at-time-of-exploitation',
+      removeDashesFromText(capitaliseText(req.sessionModel.get('pv-under-age-at-time-of-exploitation'))),
+    );
+  }
+
+  req.sessionModel.set(
+    'fr-location',
+    removeDashesFromText(capitaliseText(req.sessionModel.get('fr-location'), true)),
```
Comment (sulthan-ahmed) [inline apps/nrm/behaviours/format-answers.js]: hmm 🤔 not sure about having spaces as a value in the session, can you not just add it in Locals and just display it, see how it's done in the PDF

File: apps/nrm/behaviours/format-answers.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  );
+
+  req.sessionModel.set(
+    'any-other-pvs',
+    removeDashesFromText(capitaliseText(req.sessionModel.get('any-other-pvs'))),
+  );
```
Comment (sulthan-ahmed) [inline apps/nrm/behaviours/format-answers.js]: hmm 🤔 not sure about having spaces as a value in the session, can you not just add it in Locals and just display it, see how it's done in the PDF

Comment (jjrobothamho) [reply apps/nrm/behaviours/format-answers.js]: No problem I will do this

**Comment by sulthan-ahmed** [comment]
thanks

**Pull request closed without merging**